web/client/utils/openlayers/Icons.js

```diff
@@ -14,9 +14,11 @@ const extraMarker = markers.icons[0];
 const extraMarkerShadow = markers.icons[1];
 
 const glyphs = MarkerUtils.getGlyphs('fontawesome');
+const {isArray, isNumber} = require('lodash');
 
-const getHighlishtStyle = highlight => (highlight ? [new ol.style.Style({
+const getHighlishtStyle = ({highlight, rotation}) => (highlight ? [new ol.style.Style({
     text: new ol.style.Text({
+        rotation,
```

Comment:
I would put a default of 0 for rotation somewhere, maybe in the function argument